src/api/config/firebase.json

```diff
@@ -0,0 +1,13 @@
+{
+  "emulators": {
+    "ui": {
+      "enabled": true,
+      "host": "0.0.0.0",
+      "port": 4000
+    },
+    "firestore": {
+      "port": "8088",
```

Comment:
is there a reason why this `port` property is a string while in `ui` it is a number?

Reply:
Nice catch!  Hilariously, it's like this in the Firebase docs, so I must have copy/pasted and not noticed:

https://firebase.google.com/docs/emulator-suite/install_and_configure#port_configuration

```
{
  // Existing firebase configuration ...
  "database": {
    "rules": "database.rules.json"
  },
  "firestore" {
    "rules": "firestore.rules"
  },

  // ...

  // Optional emulator configuration. Default
  // values are used if absent.
  "emulators": {
    "firestore": {
      "port": "8080"
    },
    "ui": {
      "enabled": true,      // Default is `true`
      "port": 4000          // If unspecified, see CLI log for selected port
    },
    "auth": {
      "port": "9099"
    },
    "functions": {
      "port": "5001"
    },
    "database": {
      "port": "9000"
    },
    "pubsub": {
      "port": "8085"
    }
  }
}
```

I suspect it doesn't matter, but I'll switch to a number for all of them.